assets/component-config/vite.config.js

```diff
@@ -0,0 +1,8 @@
+import nightwatchPlugin from 'vite-plugin-nightwatch';
```

Comment:
I don't think we need to generate a vite config file at this point. In most cases, there will already be one, but if not, each plugin has its own vite config file with everything needed.